docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,356 @@
+
+          # Check that gmx_mpi exists
+          which gmx_mpi
+          cd $SLURM_SUBMIT_DIR
+          cp /apps/gromacs/water-cut1.0_GMX50_bare/1536/* .
+          scontrol show hostnames ${SLURM_JOB_NODELIST} > hostfile
+          gmx_mpi grompp -f pme.mdp -c conf.gro -p topol.top -o input.tpr
+          mpirun -n 60 -hostfile hostfile -ppn 30 gmx_mpi mdrun -notunepme -dlb yes -v -resethway -noconfout -nsteps 4000 -s input.tpr
+
+      - type: shell  # TODO: consider making bool option
+        destination: shutdown.sh
+        content: |
+          #!/bin/bash
+          if [ ! -f /etc/block_auto_shutdown ]; then
+                  touch /etc/block_auto_shutdown
+                  shutdown -h +1
+          fi
+
+  - id: spack-builder
+    source: modules/compute/vm-instance
+    use: [network-install, appsfs-install, spack-builder-startup]
+    settings:
+      name_prefix: spack-builder
+      machine_type: c2-standard-16  # TODO Consider bumping up, left low for quota
+
+- group: cluster
+  modules:
+
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - id: homefs
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.homefs_server_ip)
+      remote_mount: nfsshare
+      local_mount: /home
+      fs_type: nfs
+
+  - id: appsfs
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.appsfs_server_ip)
+      remote_mount: nfsshare
+      local_mount: /apps
+      fs_type: nfs
+
+  - id: bucket-software
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_software)
+      local_mount: /user_provided_software
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs
+
+  - id: bucket-input
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_inputs)
+      local_mount: /data_input
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs,allow_other
+
+  - id: bucket-output
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_outputs)
+      local_mount: /data_output
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs,allow_other,dir_mode=0777,file_mode=766
+
+  ### Virtual Desktop ###
+
+  - id: desktop
+    source: ./community/modules/remote-desktop/chrome-remote-desktop
+    use:
+    - network1
+    - homefs
+    - appsfs
+    - bucket-input
+    - bucket-output
+    - bucket-software
+    settings:
+      name_prefix: chrome-remote-desktop
+      install_nvidia_driver: true
+      startup_script: |
+        sudo apt-get update
+        sudo apt-get -y upgrade
+        find /user_provided_software -name vmd-1.9.*.bin.LINUXAMD64*.tar.gz -exec tar xvzf  '{}' -C . \;
+        cd vmd-1.9.*/
+        ./configure
+        cd src/
+        sudo make install
+        cd ../..
```

Comment:
`cd ../..` is technically unnecessary since it is the last command of the runner.

Reply:
Removed.